### PR TITLE
README_EXAMPLE.md template.

### PR DESCRIPTION
**Pull request by n3op2**
# Pull Request

## Checklist
- [x] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

## PR Type

Please delete options that are irrelevant.

- [x] Documentation Update

## Linked tickets

https://digicatapult.atlassian.net/browse/ENG-65

## High level description

Adding an example `EXAMPLE_README.md` file to be followed when creating README.md for the new repositories.

## Operational impact

None

### CODE REVIEW

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
Please look at this for how it currently displays.
https://github.com/digicatapult/.github/blob/e9f7a6ae9fdea2473011a7d1f044fd9a22490b90/README_EXAMPLE.md

Its pretty ugly and you need to capitalise and use correct punctuation and grammer.

File: README_EXAMPLE.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+{
+  "address": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
+  "alias": "ALICE_UPDATED"
+}
+```
+otherwise remove this section. -->
+
+<!-- if it uses TSOA/Swagger, then explain how to access it and mention that it's a swagger interface with the URL e.g: 
+Once running, the API is available at http://localhost:3000/api/docs
+-->
```
[the remark this answers is not in the document]
Comment (n3op2) [reply README_EXAMPLE.md]: Moved to the bottom of the example along with additional info

File: README_EXAMPLE.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+- Ither operations required by the service to start in development mode
+- Specifics like importing storybook components
+- If we have some seeds with regard to the development mode
+- If there are multiple scenarios. e.g. environmental persona and multi-persona, please cover both in separate paragraphs
+
+## Environment variables
+This is the list of all environment variables
+<!-- A short description of environment variables, if there are any unique cases please cover and mention where to retrieve the values -->
+- Table of environemnt variables as per the below example:
```
Comment (dblane-digicatapult) [inline README_EXAMPLE.md:72]: How are the tables generated?  Is there a tool you use/have found for doing this?

File: README_EXAMPLE.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+- database seeds if there are multiple then explain what is it for
+<!-- If we have methods for resetting the database, please cover them, it can also be a script -->
+- Resetting the database e.g: any commands like `npm run db:reset`
+
+## Service Specifics
+In this section please cover all repository specifics such as `scripts` or API specifications
+
+- Any scripts that are relevant for setting up / configuration of the service
+- API specification, please refer to the examples below
+- Ither relevant unique configurations
```
Comment (dblane-digicatapult) [inline README_EXAMPLE.md:153]:
```suggestion
- Other relevant unique configurations
```

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: README_EXAMPLE.md (hunk 2 above)
Comment (n3op2) [reply README_EXAMPLE.md:72]: added one -> https://www.tablesgenerator.com/markdown_tables that looks quite well developed with multiple options and other helpers

File: README_EXAMPLE.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+In order to form connection you would need to import DIDs that are saved in .env local file (should be mentioned in the previous paragraph). 
+#### ./scripts/<name>.sh 
+
+- A short description about each
+- Provide some examples
+- Does it take any arguments
+
+### API specification
+If service exposes some endpoints please cover them in a bit more detail here.
+<!-- ### GET /members - The address parameter identifies the user running this process, and the alias representing a more friendly name version of this. The default value of the latter is null, and is optionally set.
```
Comment (dblane-digicatapult) [inline README_EXAMPLE.md:168]:
```suggestion
<!-- #### GET /members - The address parameter identifies the user running this process, and the alias representing a more friendly name version of this. The default value of the latter is null, and is optionally set.
```

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**